editor/src/clj/editor/scene.clj

```diff
@@ -600,6 +600,22 @@
             union-aabb
             (:children scene))))
 
+(declare active-scene-view)
+(declare frame-selection)
+
+(defn- reframe [scene scene-aabb active-view]
+  (when (:node-id scene)
+    (let [node-id (:node-id scene)
+          prev-aabb (g/user-data node-id :prev-scene-aabb)
+          reframe? (if prev-aabb
+                     (and (geom/predefined-aabb? prev-aabb)
+                          (not (geom/predefined-aabb? scene-aabb)))
```

Comment:
here we check if prev. saved AABB one of predefined and new AABB isn't then we reframe camera view on an added object